README.md

```diff
@@ -1,8 +1,14 @@
-# Blueflood Datasource
+# Blueflood plugin for grafana
 
-Blueflood is a multi-tenant distributed metric processing system created by engineers at Rackspace. It is used in production by the Rackspace Monitoring team to process metrics generated by their monitoring systems.
-Blueflood is capable of ingesting, rolling up and serving metrics at a massive scale.
+This repo contains transpiled code of the [blueflood-grafana](https://github.com/rackerlabs/blueflood-grafana) plugin source code. 
+The code in this repo can be directly installed as a blueflood plugin for grafana. 
+   
+For more information on how to install the plugin, refer [blueflood-grafana](https://github.com/rackerlabs/blueflood-grafana).
+   
+## Contributions
```

Comment:
This plugin is exact replica of graphite plugin, which is complete and looked after by Raintank.